src/OrchardCore.Modules/OrchardCore.ContentFields/Assets/js/trumbowyg.theme.js

```diff
@@ -0,0 +1,17 @@
+document.addEventListener('DOMContentLoaded', function () {
+    function setTheme() {
+        var isDark = getPreferredTheme() == darkThemeName;
+
+        document
+            .querySelectorAll('.trumbowyg-wrapper')
+            .forEach((element) =>
+                element.classList.toggle('trumbowyg-dark', isDark)
```

Comment:
```suggestion
        document
            .querySelectorAll('.trumbowyg-wrapper')
            .forEach((element) =>
                element.classList.toggle('trumbowyg-dark', isDarkMedia())
```

Reply:
I don't think `isDarkMedia()` will be enough here, right?

Reply:
It should be all you need. 

https://github.com/OrchardCMS/OrchardCore/blob/cd442781ced1c6c4e505bdf097ac0937051916f1/src/OrchardCore.Themes/TheAdmin/Assets/js/header/global/constants.js#L6

Reply:
It should first check the user selection, then fall back to device settings. This is how `getPreferredTheme()` works.